src/lib.rs

```diff
@@ -8,7 +8,7 @@ pub use crate::{
     fetch::{Method, Request},
     routing::{push_route, Url},
     util::{body, cookies, document, error, history, html_document, log, update, window},
-    vdom::{App, AppBuilder},
+    vdom::{App, AppBuilder, MountType},
```

Comment:
Why did you add it here instead of `prelude`?

Reply:
Oh, oops. My bad. Must have been tired yesterday.